packages/flutter_arc_text/lib/src/arc_text.dart

```diff
@@ -52,6 +53,8 @@ class ArcText extends StatelessWidget {
   /// Text placement relative to circle with the same [radius].
   final Placement placement;
 
+  final PainterDelegate paint;
```

Comment:
I guess it wouldn't cost that much to name it `painterDelegate` and it would avoid possible confusions.